src/target/adiv5.c

```diff
@@ -1079,10 +1077,10 @@ const void *adiv5_pack_data(const uint32_t dest, const void *const src, uint32_t
 	return (const uint8_t *)src + (1 << align);
 }
 
-void advi5_mem_read_bytes(adiv5_access_port_s *ap, void *dest, uint32_t src, size_t len)
+void advi5_mem_read_bytes(adiv5_access_port_s *const ap, void *dest, uint32_t src, size_t len)
```

Comment:
Might as well `const` the other parameters as much as possible if doing this.

Reply:
I think this slipped because I was commiting a  couple concurrent change sets in parallel, I'll do that

Reply:
Nevermind, looks like I didn't miss anything, those parameters are modified in the function body